src/AutorestSwift/Models/Complex/ParameterType.swift

```diff
@@ -151,6 +151,62 @@ enum ParameterType: Codable {
     internal func belongsInOptions() -> Bool {
         return common.belongsInOptions()
     }
+
+    /**
+     Convert the type into String format in Swift
+     */
+    public func formatValue(value: String) -> String {
```

Comment:
Same comment on docstring, but this is an instance where you would use either of these signatures:

```swift
public func format(value: String) -> String {...}
public func formatValue(_ value: String) -> String {...}
```

For consistency with the other instance of this method, I would recommend the second option.